src/components/raycaster.js

```diff
@@ -38,7 +38,7 @@ module.exports.Component = registerComponent('raycaster', {
     direction: {type: 'vec3', default: {x: 0, y: 0, z: -1}},
     enabled: {default: true},
     far: {default: 1000},
-    interval: {default: 0},
+    interval: {default: 100},
```

Comment:
I don't think you meant to revert this change.

Reply:
I'll do in another commit, i think 100 was appropriate for perf